mapbox_gl_web/pubspec.yaml

```diff
@@ -21,7 +21,7 @@ dependencies:
       url: https://github.com/tobrun/flutter-mapbox-gl.git
       path: mapbox_gl_platform_interface
   mapbox_gl_dart: ^0.2.1
-  image: ^3.0.0
+  image: ^4.0.15
```

Comment:
change this to `image: '>=3.0.0 <5.0.0'` so it keeps working with 3.x as well - 4.x seems to be fine from what i can tell

Reply:
@Cortex78 I verified that @felix-ht's suggestion works. Thoughts?